frontend/src/pages/instance/components/InstanceLogs.vue

```diff
@@ -117,7 +117,9 @@ export default {
                     entries.log.forEach(l => {
                         const d = new Date(parseInt(l.ts.substring(0, l.ts.length - 4)))
                         l.date = `${d.toLocaleDateString()} ${d.toLocaleTimeString()}`
-                        if (typeof l.msg !== 'string') {
+                        if (l.msg === undefined) {
```

Comment:
```suggestion
                        if (typeof l.msg === `undefined`) {
```

undefined is mutable